target/linux/ipq807x/files/arch/arm64/boot/dts/qcom/ipq8074-sxr80.dts

```diff
@@ -0,0 +1,643 @@
+
+		led_front_blue: front-blue {
+			label = "blue:front";
+			gpios = <&tlmm 33 GPIO_ACTIVE_LOW>;
+		};
+
+		led_front_green: front-green {
+			label = "green:front";
+			gpios = <&tlmm 29 GPIO_ACTIVE_LOW>;
+		};
+
+		led_front_red: front-red {
+			label = "red:front";
+			gpios = <&tlmm 31 GPIO_ACTIVE_LOW>;
+		};
+
+		led_front_white: front-white {
+			label = "white:front";
+			gpios = <&tlmm 26 GPIO_ACTIVE_LOW>;
+		};
+
+		led_power_green: power-green {
+			label = "green:power";
+			gpios = <&tlmm 21 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		led_power_red: power-red {
+			label = "red:power";
+			gpios = <&tlmm 22 GPIO_ACTIVE_LOW>;
+			panic-indicator;
+		};
+
+	};
+};
+
+&tlmm {
+	mdio_pins: mdio-pins {
+		mdc {
+			pins = "gpio68";
+			function = "mdc";
+			drive-strength = <8>;
+			bias-pull-up;
+		};
+
+		mdio {
+			pins = "gpio69";
+			function = "mdio";
+			drive-strength = <8>;
+			bias-pull-up;
+		};
+	};
+
+	i2c_0_pins: i2c-0-pinmux {
+		mux {
+			pins = "gpio42", "gpio43";
+			function = "blsp1_i2c";
+			drive-strength = <8>;
+			bias-disable;
+		};
+	};
+
+	leds_pins: leds_pinmux {
+		led_power_green {
+			pins = "gpio21";
+			function = "gpio";
+			drive-strength = <8>;
+			bias-pull-down;
+		};
+		led_power_red {
+			pins = "gpio22";
+			function = "gpio";
+			drive-strength = <8>;
+			bias-pull-down;
+		};
+		led_white {
+			pins = "gpio26";
+			function = "gpio";
+			drive-strength = <8>;
+			bias-pull-down;
+		};
+		led_green {
+			pins = "gpio29";
+			function = "gpio";
+			drive-strength = <8>;
+			bias-pull-down;
+		};
+		led_red {
+			pins = "gpio31";
+			function = "gpio";
+			drive-strength = <8>;
+			bias-pull-down;
+		};
+		led_blue {
+			pins = "gpio33";
+			function = "gpio";
+			drive-strength = <8>;
+			bias-pull-down;
+		};
+	};
+
+
+};
+
+&blsp1_uart5 {
+	status = "okay";
+};
+
+&blsp1_i2c2 {
+	pinctrl-0 = <&i2c_0_pins>;
+	pinctrl-names = "default";
+
+	status = "okay";
+	
+	tlc59208f@27 {
+		#address-cells = <1>;
+		#size-cells = <0>;
+		compatible = "ti,tlc59108";
+		reg = <0x27>;
+		
+		led0@0 {
+			label = "orbi:led0:rgb";
+			reg = <0x0>;
+			linux,default_trigger = "default-on";
+		};
+		led1@1 {
+			label = "orbi:led1:rgb";
+			reg = <0x1>;
+			linux,default_trigger = "default-on";
+		};
+		led2@2 {
+			label = "orbi:led2:rgb";
+			reg = <0x2>;
+			linux,default_trigger = "default-on";
+		};
+		led3@3 {
+			label = "orbi:led3:rgb";
+			reg = <0x3>;
+			linux,default_trigger = "default-on";
+
+		};
+	};
+};
+
+&prng {
+	status = "okay";
+};
+
+&cryptobam {
+	status = "okay";
+};
+
+&crypto {
+	status = "okay";
+};
+
+&qpic_bam {
+	status = "okay";
+};
+
+&qpic_nand {
+	status = "okay";
+
+	nand@0 {
+		reg = <0>;
+		nand-ecc-strength = <4>;
+		nand-ecc-step-size = <512>;
+		nand-bus-width = <8>;
+
+		partitions {
+			compatible = "qcom,smem-part";
+		};
+	};
+};
+
+&mdio {
+	status = "okay";
+
+	pinctrl-0 = <&mdio_pins>;
+	pinctrl-names = "default";
+
+	reset-gpios = <&tlmm 25 1>;
```

Comment:
Please use the GPIO_ACTIVE_HIGH and GPIO_ACTIVE_LOW defines here instead.

Reply:
Also, you cant have reset-gpios here and then under the PHY itself

Reply:
IIRC this is the "workaround" for the PHY, the ones in the PHY are commented out as multiple reset-gpios caused issues. If that's fixed the ones in the PHY can be turned on again

Reply:
I know, but its not correct.
I have commented on the QCA8081 what to do

Reply:
Fixed